JULIERAJ/KIDS-FIRST-v2
# Design Updates, Error Messages, and Form Field Adjustments

This PR enhances the sign-up form based on the latest Figma design specifications. 
Design Update

Figma Alignment: Updated the sign-up form to match the latest Figma design version.
Styling Enhancements: Improved the visual styling for inputs, buttons, and labels.

Form Field Modifications

Added Fields: Introduced input fields for First Name and Last Name.
Removed Field: Eliminated the Password Confirmation field to streamline the user registration process.

Error Message Implementation

Inline Error Messages: Added inline error messages for invalid inputs, providing immediate feedback to users.
Visual Feedback: Incorporated visual feedback for errors, such as red borders and error icons next to problematic fields.

Backend Update:

User Model: Updated the user model to include firstName and lastName fields.

Screenshots:

Figma Design:
![Screenshot 2024-06-14 at 10 27 39 PM](https://github.com/JULIERAJ/KIDS-FIRST-v2/assets/47821694/1f2c2f29-e4de-4f8d-abf3-c52730e86f92)


Before Update

![Screenshot 2024-06-14 at 10 13 04 PM](https://github.com/JULIERAJ/KIDS-FIRST-v2/assets/47821694/dd5fd3bf-c395-42a7-abfc-71d3a6859304)

After Update 
[
![Screenshot 2024-06-14 at 9 57 15 PM](https://github.com/JULIERAJ/KIDS-FIRST-v2/assets/47821694/8a888e05-704a-468f-a676-2d77f3711716)
](url)

Testing Instructions
Navigate to the Sign-Up Page: Confirm the form layout and styling match the Figma design.
Input Validation: Test the new fields (First Name, Last Name) for proper validation.
Error Handling: Verify error messages appear correctly when invalid data is entered.
Form Submission: Ensure the form is submitted correctly without the password confirmation field. 

Note. 
Please, during testing, disregard the console error related to App.jsx:33. This issue is unrelated to the sign-up form and is being addressed separately.
